api/types/constants.go

```diff
@@ -297,6 +297,10 @@ const (
 )
 
 const (
+	// TeleportNamespace is used as the namespace prefix for any
+	// labels defined by teleport
+	TeleportNamespace = "teleport.dev/"
```

Comment:
I think we use this label quite a bit throughout the code. If we're introducing a constant for it, I feel like we should go ahead and use it in all places that currently use `teleport.dev`. Otherwise you'd have to remember to change it everywhere. So ideally the only place "teleport.dev" string should appear verbatim is in this constant definition.